server/odc-service/src/main/java/com/oceanbase/odc/service/session/factory/OBConsoleDataSourceFactory.java

```diff
@@ -172,10 +172,6 @@ public static Map<String, String> getJdbcParams(@NonNull ConnectionConfig connec
         } else {
             jdbcUrlParams.put("useSSL", "false");
         }
-        DialectType type = connectionConfig.getDialectType();
-        if (type != null && type.isOceanbase()) {
-            jdbcUrlParams.put("enableFullLinkTrace", "true");
```

Comment:
Why removing this block of logic

Reply:
The jdbc parameters related to the data source are all removed to `ConnectionExtensionPoint.appendDefaultJdbcUrlParameters` method of connect-plugin